### DISCUSSION TITLE
Question about the usability of the api

### DISCUSSION THREAD

**Discussion by kypanz** [remotive-com/remote-jobs-api#11] [General]
Hello guys, sorry about my English but i have a question

What is the finally usability of this api ?
i am asking because if i cant post the jobs that is returned from an api that is called literally "remote-jobs-api"

what is the propouse of the api ?

i am trying to build a global searcher too, so i respect the terms of services but i am curious about what is the idea of the api if i cant show the returned data anywhere

some ideas ?

**Comment by adriencombourieu** [comment]
Hi Kypanz,

You can use the public API for example : 
- for your own job search, if you are looking for a job
- to display remotive jobs on your own website. In that case:

1. You must link the jobs to the URL on remotive.com
2. The jobs from Remotive must be accessible freely to your users, without payment nor account creation/login

If you intend to make a commercial use out of Remotive's data, then contact us (hello@remotive.com) to discuss the terms.

**Comment by kypanz** [comment]
Hello, thanks for the answer

Just for now i gonna use it to integrate into a searcher for jobs, for now is not paid so i think i can used

All the links is gonna be redirected to the official website of remotive and in the future if is pay i gonna contact you

thanks for the answer and have a nice weekend :)
